lib/sendgrid_actionmailer.rb

```diff
@@ -302,22 +291,17 @@ def perform_send_request(email)
       result
     end
 
-    # Returns a new hash with the results of running the block once for every
-    # key. This method does not change the values.
-    #
-    # Hash#transform_keys was introduced in Ruby 2.5. We're reimplementing it
-    # here to maintain backwards compatibility with older Rubies.
-    #
-    # If the method is available on Hash already, we'll use that implementation.
-    def transform_keys(hash, block = Proc.new)
-      return hash.transform_keys do |key|
-        block.call(key)
-      end if hash.respond_to?(:transform_keys)
-
-      hash.map do |key, value|
-        new_key = yield(key)
-        [new_key, value]
-      end.to_h
+    # Recursive key transformation based on Rails deep_transform_values
+    def self.transform_keys(object, &block)
+      #p object
```

Comment:
Looks like a bit of debugging code snuck it's way in here :)

Reply:
Thanks, removed.